app/src/main/java/com/kickstarter/viewmodels/RewardCardUnselectedViewHolderViewModel.kt

```diff
@@ -1,76 +1,66 @@
-        fun buttonCTA() : Observable<Int>
+        /** Emits a boolean that determines if the card is clickable. */
+        fun isClickable(): Observable<Boolean>
 
-        /** Emits a boolean that determines if the pledge button should be enabled. */
-        fun buttonEnabled() : Observable<Boolean>
+        /** Emits the alpha value for the card issuer image. */
+        fun issuerImageAlpha(): Observable<Float>
+
+        /** Emits the text color resource ID for the last four copy. */
+        fun lastFourTextColor(): Observable<Int>
 
         /** Emits a boolean that determines if the not available copy should be visible. */
         fun notAvailableCopyIsVisible(): Observable<Boolean>
 
         /** Emits a string representing the project's country when the card is not accepted. */
         fun projectCountry(): Observable<String>
+
+        /** Emits a boolean that determines if the select image should be visible. */
+        fun selectImageIsVisible(): Observable<Boolean>
     }
 
     class ViewModel(environment: Environment) : BaseRewardCardViewHolderViewModel.ViewModel(environment), Inputs, Outputs  {
         val inputs: Inputs = this
         val outputs: Outputs = this
 
-        private val buttonCTA = BehaviorSubject.create<Int>()
-        private val buttonEnabled = BehaviorSubject.create<Boolean>()
+        private val isClickable = BehaviorSubject.create<Boolean>()
+        private val issuerImageAlpha = BehaviorSubject.create<Float>()
+        private val lastFourTextColor = BehaviorSubject.create<Int>()
         private val notAvailableCopyIsVisible = BehaviorSubject.create<Boolean>()
         private val projectCountry = BehaviorSubject.create<String>()
+        private val selectImageIsVisible = BehaviorSubject.create<Boolean>()
 
         init {
-            val card = this.cardAndProject
-                    .map { it.first }
 
             val project = this.cardAndProject
                     .map { it.second }
 
-            val backing = project
-                    .map { it.backing() }
-
-            val isBackingPaymentSource = backing
-                    .compose<Pair<Backing?, StoredCard>>(combineLatestPair(card))
-                    .map { backingAndCard -> backingAndCard.first?.let { b -> b.paymentSource()?.let { it.id() == backingAndCard.second.id() } }?: false }
-
             val allowedCardType = this.cardAndProject
                     .map { ProjectUtils.acceptedCardType(it.first.type(), it.second) }
 
-            val isBackingPaymentAndAllowedType = isBackingPaymentSource
-                    .compose<Pair<Boolean, Boolean>>(combineLatestPair(allowedCardType))
+            allowedCardType
+                    .compose(bindToLifecycle())
+                    .subscribe(this.isClickable)
 
-            isBackingPaymentAndAllowedType
-                    .map { !it.first && it.second }
+            allowedCardType
+                    .map { if (it) 1.0f else .5f }
```

Comment:
Is this to fade the card image if there the card is disabled? I don't know if there are constants elsewhere in the app for this or if a similar behavior is found elsewhere?

Reply:
Yeah! No constants, this is our first time disabling an image in this way (previously we disabled the button).

In the activity feed, we make project images `.2` alpha but that's also just a hardcoded number.

Reply:
Ok lets keep it the same as the Figma designs